go-apps/meep-gis-engine/server/README.md

```diff
@@ -0,0 +1,25 @@
+# Go API Server for server
```

Comment:
This title looks weird... maybe a byproduct of the generating toolchain or an error in the source swagger file.